.eslintrc.json

```diff
@@ -2,6 +2,7 @@
   "root": true,
   "extends": ["airbnb-base", "plugin:jest/recommended", "plugin:node/recommended", "prettier"],
   "plugins": ["jest", "node"],
+  "ignorePatterns": ["website/**/*"],
```

Comment:
I ignored the website folder for eslint because it has rules to check the installed dependencies of the project. Since I haven't added the website to Lerna's configuration, it is currently erroring. Let me know if you want to do it differently.